content/sensu-go/5.10/installation/install-sensu.md

```diff
@@ -160,7 +160,7 @@ Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.10.2/se
 Start the installation wizard.
```

Comment:
Running the command with `/qn` actually causes the installation to proceed without a wizard. I think we should either change this text or describe the `/qn` flag as an alternate instruction